container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/shared_helpers/shared_helpers.go

```diff
@@ -459,6 +476,69 @@ func GetEngineAndLogsComponentsNetwork(
 	return targetNetwork, nil
 }
 
+func DumpContainers(ctx context.Context, dockerManager *docker_manager.DockerManager, containers []*types.Container, outputDirpath string) error {
```

Comment:
copied this over from container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend_enclave_functions.go